lib/ably/auth.rb

```diff
@@ -25,21 +27,21 @@ module Ably
   #   @return [Hash] Default token params used for token requests, see {#request_token}
   #
   class Auth
-    include Ably::Modules::Conversions
-    include Ably::Modules::HttpHelpers
+    include ::Ably::Modules::Conversions
+    include ::Ably::Modules::HttpHelpers
 
     # Default capability Hash object and TTL in seconds for issued tokens
     TOKEN_DEFAULTS = {
       renew_token_buffer: 10 # buffer to allow a token to be reissued before the token is considered expired (Ably::Models::TokenDetails::TOKEN_EXPIRY_BUFFER)
     }.freeze
 
-    API_KEY_REGEX = /^[\w-]{2,}\.[\w-]{2,}:[\w-]{2,}$/
+    API_KEY_REGEX = /^[\w-]{2,}\.[\w-]{2,}:[\w-]{2,}$/.freeze
```

Comment:
Is is an offense? I'm pretty sure regex freeze is redundant
https://github.com/rubocop/rubocop/issues/6331

Reply:
Fixed in 47f3bf7494f00fe6aaeaec2f3230f4f2737d3c3b